component/webui/src/main/java/org/exoplatform/social/webui/space/UISpaceSettings.java

```diff
@@ -48,7 +48,7 @@ public UISpaceSettings(String name) throws Exception {
     //
     addUIFormInput(new UIFormStringInput(SPACE_DISPLAY_NAME, SPACE_DISPLAY_NAME, "").
                    addValidator(MandatoryValidator.class).
-                   addValidator(ExpressionValidator.class, "^([\\p{L}\\s\\d\']+[\\s]?)+$", MSG_INVALID_SPACE_NAME).
+                   addValidator(ExpressionValidator.class, "^([\\p{L}\\d\']+[\\s]?)+$", MSG_INVALID_SPACE_NAME).

```

Comment:
I tested this regex "^[\p{L}\d\s']+$" but the problem is not solved (the name with 2 spaces is accepted at creation).